tests/python/unittest/test_gluon_rnn.py

```diff
@@ -171,6 +171,54 @@ def test_stack():
     assert outs == [(10, 100), (10, 100), (10, 100)]
 
 
+def test_hybridstack():
```

Comment:
Do we have a unit test for the NC layout bug you fixed?

Reply:
There is NTC unroll in this test